Lib/test/test_curses.py

```diff
@@ -1364,11 +1364,9 @@ def test_move_left(self):
         self.mock_win.reset_mock()
         self.textbox.do_command(curses.KEY_LEFT)
         self.mock_win.move.assert_called_with(1, 0)
-        self.textbox.do_command(curses.KEY_RIGHT)
-        self.mock_win.move.assert_called_with(1, 2)
```

Comment:
Hmm, I wonder if the intent here was to test that moving twice works?

Reply:
I've added `test_move_left_and_right`. It won't hurt :)